rest/stats_context.go

```diff
@@ -16,6 +16,7 @@ import (
 // Group the stats related context that is associated w/ a ServerContext into a struct
 type statsContext struct {
 	statsLoggingTicker *time.Ticker
+	statsLoggingStop   chan struct{}
```

Comment:
We use the convention 'terminator' for naming this type of channel throughput Sync Gateway - I think it makes sense to follow that here as well.